vitrina/cms/templates/pages/page_with_side_menu.html

```diff
@@ -0,0 +1,76 @@
+    <meta name="description" content="{% block pageDescription %}Atvirų duomenų portalas - tai vieningas prieigos taškas prie visų Lietuvoje atvertų duomenų rinkinių ir suteikiantis technologines priemones atvirų duomenų tiekėjams parengti ir publikuoti atvertų duomenų rinkinių metaduomenis.{% endblock %}">
+    <meta name="keywords" content="{% block pageKeywords %}atviri duomenys, duomenų rinkiniai, atvirų duomenų portalas{% endblock %}">
+    <meta name="author" content="{% block pageAuthor %}Informacinės visuomenės plėtros komitetas{% endblock %}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta property="og:title" content="Lietuvos atvirų duomenų portalas{% block pageOgTitle %}{% endblock %}" />
+    <meta property="og:type" content="{% block pageOgType %}website{% endblock %}" />
+    <meta property="og:description" content="{% block pageOgDescription %}Atvirų duomenų portalas - tai vieningas prieigos taškas prie visų Lietuvoje atvertų duomenų rinkinių ir suteikiantis technologines priemones atvirų duomenų tiekėjams parengti ir publikuoti atvertų duomenų rinkinių metaduomenis.{% endblock %}" />
+
+    <link type="text/css" href="{% static 'css/bundle.css' %}" rel="stylesheet" media="screen">
+    <script defer src="{% static 'js/bundle.js' %}"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+    {% include "component/favicon.html" %}
+    {% render_block "css" %}
+</head>
+<body {% if stagingEnv %}class="contains-warning-border"{% endif %}>
+{% block warning %}
+    {% if stagingEnv %}
+        <div id="top-bar" class="is-warning-bar">
+            <p>"testingEnvironment"</p>
+        </div>
+    {% endif %}
+{% endblock %}
+{% block messages %}
+    {% if messages %}
+        <ul class="messages">
+            {% for message in messages %}
+                {% if message.tags %} <li class="{{ message.tags }}" id="message-list-{{ forloop.counter }}"> {% else %} <li> {% endif %}
+                    {{ message }}
+                    <span class="is-pulled-right message-close-button" onClick="HideMessage({{ forloop.counter }})">X</span>
+                </li>
+            {% endfor %}
+        </ul>
+    {% endif %}
+{% endblock messages %}
+<header>
+    {% include "component/navbar.html" %}
+</header>
+
+{% block breadcrumbs %}
+    <nav class="breadcrumb has-arrow-separator" aria-label="breadcrumbs">
+        <div class="container needs-padding">
+            <ul class="p-t-sm">
+                <li><a href="{% block parent_url %}{% endblock %}"><span>{% block parent_title %}{% endblock %}</span></a></li>
+            </ul>
+            <span class="breadcrumbs-current is-size-5-mobile">{% block current_title %}{% page_attribute "page_title" %}{% endblock %}</span>
+        </div>
+    </nav>
+{% endblock %}
+
+<div class="container full-h needs-padding">
+    {% cms_toolbar %}
+    <div class="columns">
+        {% placeholder "side_menu" %}
```

Comment:
For some reason if I select this template from the UI, sidebar does not show up:

![image](https://user-images.githubusercontent.com/297583/205019907-b8bc7467-20ab-4d43-a02a-a5118d03f060.png)

As you can see in the image, template with side menu is selected, but side manu dos not appear.

Reply:
Side menu is added with plugin. I couldn't find a way to add it directly to the template, because django cms only provides template tags to get page title, menu title, slug, ect. It doesn't give hierarchy which is needed for menu, that's why I added SideMenuPlugin. 
So when user choses template with side menu, what django does, it creates template with two blocks: one for side menu and another one for body, then user has to add side menu plugin to that block (it can be done by pressing cms toolbar button at the top right corner).